src/thorin/continuation.cpp

```diff
@@ -139,13 +139,42 @@ const Param* Continuation::mem_param() const {
 }
 
 const Param* Continuation::ret_param() const {
+    if (is_intrinsic()) {
```

Comment:
solved by https://github.com/AnyDSL/thorin/commit/a0017801999bd72b7bbd5413d235c1b9595fb8b1

Reply:
Removed the offending commit.